package.json

```
@@ -7,6 +7,9 @@
     "build": "browserify example/example.js -g uglifyify -o bundle.js",
     "convert": "node convert.js > fox.json"
   },
+  "files": [
+    "bundle.js"
```

`index.js` is the "main" module (see the `main` field in `package.json`), so it is certainly required. `fix.json` is required because it's referenced by `index.js`. Maybe this is it though? :thinking: 

```suggestion
    "bundle.js",
    "index.js",
    "fox.json"
```

---

Oof, sorry, not my best work.

Upon further investigation (that should've been done earlier), `bundle.js` is (as you noted elsewhere) not even used by consumers. All we need is `fox.json` and `index.js`, and the latter is included by default, regardless of the files property.

So, we only need to specify `fox.json`, and that's done in: f53be13

---

It still might be worth including `bundle.js` if this package is meant to be used directly by websites (e.g. similar to `@metamask/onboarding`, which includes a bundle that can be used directly via a service like unpkg).

I wasn't sure if this package was used that way 🤷‍♂️ or if it was meant to be.

---

>and the latter is included by default

Huh - TIL

---

> I wasn't sure if this package was used that way 🤷‍♂️ or if it was meant to be.

Hm, same, but I reckon you're right that we should add it back, which I did here: I wasn't sure if this package was used that way 🤷‍♂️ or if it was meant to be.